### ISSUE TITLE
Multiple Instruments

### ISSUE THREAD

**Issue by jazzyeagle** [jazzyeagle/improvibot#2]
There should be multiple neural networks based on category of output (e.g. melody, harmony, bass, drums, etc.).  The melody track would be generated first, and the output of the former (in terms of the generated notes, not the softmax'd weights) would be the input of the latter to generate the entire song.
